platform-includes/debug-symbols-apple/_default.mdx

```diff
@@ -1,5 +1,11 @@
 Sentry requires dSYMs (debug information files) to symbolicate your stack traces. The symbolication process unscrambles the stack traces to reveal the function, file names, and line numbers of the crash.
 
+You can also upload your code for source context. This feature allows Sentry to display snippets of your code next to the event stack traces.
+
+<OnboardingOptionButtons
+  options={[{ id:"source-context", checked: false }]}
+/> 
```

Comment:
`h`: I think this should be directly above the code snippets. The experience is quite confusing to me now as it's I have no clue what the checkbox is doing, cause I don't see the code snippets.

![CleanShot 2024-11-14 at 16 10 11@2x](https://github.com/user-attachments/assets/10524220-5931-4427-afe9-dabe5f3d47b3)

Reply:
@brustolin, I think it would be better to somehow make the button non sticky. When I navigate down to the SentryFastlane plugin, I now see the button but it misses explanation.
![CleanShot 2024-11-18 at 15 56 35@2x](https://github.com/user-attachments/assets/e46dc6a6-71cf-4bbe-b3da-612de2e21737)

Reply:
But It affects all the code snippets in the page. And since the entire page is about uploading debug information, I think this makes sense. 

Its like this in all other pages that this component is used.